src/App.js

```diff
@@ -26,39 +28,26 @@ class App extends Component {
               <a href="#">Auth0 - React</a>
             </Navbar.Brand>
             <Button
-              bsStyle="primary"
-              className="btn-margin"
-              onClick={this.goTo.bind(this, 'home')}
-            >
-              Home
-            </Button>
-            {
-              !isAuthenticated() && (
-                  <Button
-                    bsStyle="primary"
-                    className="btn-margin"
-                    onClick={this.login.bind(this)}
-                  >
-                    Log In
-                  </Button>
-                )
-            }
-            {
-              isAuthenticated() && (
-                  <Button
-                    bsStyle="primary"
-                    className="btn-margin"
-                    onClick={this.logout.bind(this)}
-                  >
-                    Log Out
-                  </Button>
-                )
-            }
+              {...{
```

Comment:
Instead of using html to set props on an element, I prefer to extend an object. I feel it looks cleaner, but I don't know if there are any performance benefits you get by doing it the "react-y way"